Editor/Scripts/MeshSyncServerInspector.cs

```diff
@@ -1,12 +1,81 @@
-using Unity.FilmInternalUtilities;
+using System.Collections.Generic;
 using Unity.FilmInternalUtilities.Editor;
 using UnityEditor;
 using UnityEngine;
 
 namespace Unity.MeshSync.Editor  {
 [CustomEditor(typeof(MeshSyncServer))]
-internal class MeshSyncServerInspector : BaseMeshSyncInspector   {
-    
+[InitializeOnLoad]
+internal class MeshSyncServerInspector : BaseMeshSyncInspector {
+#if MESHSYNC_PROBUILDER_SUPPORT
+    static System.Reflection.FieldInfo versionInfo = typeof(UnityEngine.ProBuilder.ProBuilderMesh).GetField("m_VersionIndex", System.Reflection.BindingFlags.NonPublic | System.Reflection.BindingFlags.Instance);
+
+    static Dictionary<UnityEngine.ProBuilder.ProBuilderMesh, ushort> versionIndices = new Dictionary<UnityEngine.ProBuilder.ProBuilderMesh, ushort>();
+
+    static MeshSyncServerInspector()
+    {
+        BaseMeshSync.ProBuilderBeforeRebuild += ProBuilderBeforeRebuild;
+        BaseMeshSync.ProBuilderAfterRebuild += ProBuilderAfterRebuild;
```

Comment:
New Callbacks will be registered everytime we show the inspector of MeshSyncServer.
Is this expected ?

Reply:
This is the static constructor, it's only called once and it has the [InitializeOnLoad] attribute so it's also registered before it shows.